s3same/iam.py

```diff
@@ -0,0 +1,75 @@
+from botocore.exceptions import ClientError
+
+IAMName = 's3same_travis'
+
+def _policy_string(bucket):
+    return """{
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Action": [
+            "s3:ListBucket"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+            "arn:aws:s3:::%(bucket)s"
+          ]
+        },
+        {
+          "Action": [
+            "s3:PutObject",
+            "s3:PutObjectAcl"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+            "arn:aws:s3:::%(bucket)s/*"
+          ]
+        }
+      ]
+    }""" % dict(bucket=bucket)
```

Comment:
The `%` operator as a string formatter has been deprecated in Python 3. I'm not sure we have any code standards surrounding it, but even in Python 2.7, [string.format()](https://docs.python.org/2/library/string.html#string-formatting) is [designed to replace](https://www.python.org/dev/peps/pep-3101/) the `%` operator. The old operator may actually be removed one of these centuries, and it's generally a good idea to use the new method unless backwards-compatibility with Python <2.6 is necessary.

Unfortunately, this is actually a terrible case for the improvements that `string.format()` offers. You can replace `%(bucket)s` with `{bucket}` and `% dict(bucket=bucket)` with `.format(bucket=bucket)`. But since the braces are now special characters, the literal braces you include have to be escaped by doubling them.